docs/management_api_v2.yml

```diff
@@ -0,0 +1,264 @@
+                description: List of ordered sort criterias
+                items:
+                  $ref: '#/definitions/SortCriteria'
+
+      responses:
+        200:
+          description: Successful response.
+          headers:
+            Link:
+              type: string
+              description: |
+                Standard header, used for page navigation.
+
+                Supported relation types are 'first', 'next' and 'prev'.
+            X-Total-Count:
+              type: string
+              description: Custom header indicating the total number of devices for the given query parameters
+          schema:
+            title: ListOfDevices
+            type: array
+            items:
+              $ref: '#/definitions/Device'
+          examples:
+            application/json:
+              - id: "291ae0e5956c69c2267489213df4459d19ed48a806603def19d417d004a4b67e"
+                attributes:
+                  - name: "ip_addr"
+                    value: "1.2.3.4"
+                    description: "IP address"
+                  - name: "mac_addr"
+                    value: "00.01:02:03:04:05"
+                    description: "MAC address"
+                  - name: "ports"
+                    value:
+                      - "8080"
+                      - "8081"
+                    description: "Open ports"
+                updated_ts: "2016-10-03T16:58:51.639Z"
+              - id: "76f40e5956c699e327489213df4459d1923e1a806603def19d417d004a4a3ef"
+                attributes:
+                  - name: "mac"
+                    value: "00:01:02:03:04:05"
+                    description: "MAC address"
+                updated_ts: "2016-10-04T18:24:21.432Z"
+        400:
+          description: Missing or malformed request parameters. See error for details.
+          schema:
+            $ref: '#/definitions/Error'
+        500:
+          description: Internal error.
+          schema:
+            $ref: '#/definitions/Error'
+
+definitions:
+  Attribute:
+    description: Attribute descriptor.
+    type: object
+    required:
+      - name
+      - scope
+      - value
+    properties:
+      name:
+        type: string
+        description: |
+            A human readable, unique attribute ID, e.g. 'device_type', 'ip_addr', 'cpu_load', etc.
+      scope:
+        type: string
+        description: |
+            The scope of the attribute.
+
+            Scope is a string and acts as namespace for the attribute name.
+      description:
+        type: string
+        description: Attribute description.
+      value:
+        type: string
+        description: |
+            The current value of the attribute.
+
+            Attribute type is implicit, inferred from the JSON type.
+
+            Supported types: number, string, array of numbers, array of strings.
+            Mixed arrays are not allowed.
+    example:
+      application/json:
+        name: "cpu_load"
+        scope: "metrics"
```

Comment:
iirc all scopes designed by eysteing are: inventory, identity, system, custom. maybe let's use `inventory`? or something from this list, the user might be confused otherwise

Reply:
Ok, fixed.